### ISSUE TITLE
💸 SEMrush - Reduced number of crawled pages

### ISSUE THREAD

**Issue by amankumarrr** [SSWConsulting/SSW.Website#2974]
@wicksipedia @Marxoz 

**Description**
Recently, we observed a significant drop in the SEMrush audit report. The audit is able to crawl an average of 3600 pages, but we have approximately 3600+ pages just for the /rules section.

**Proposed Solution**

The SEMrush audit should be configured to crawl all relevant sections of the site, including:

- Main website
- `/archive`
- `/rules`
- `/people`

- [x] Investigate and fix the issue
- [x] Investigate if the crawl is impacting our SEO

**Comment by amankumarrr** [comment]
@wicksipedia @bradystroud @jeoffreyfischer @Calinator444 

Based on the sitemaps for each site, SEMrush should be able to crawl the following number of pages:

- `ssw.com.au` -> 402 pages
- `ssw.com.au/people` -> 584 pages
- `ssw.com.au/rules` -> 3,689 pages
- `ssw.com.au/archives` -> 1,011 pages

In total, a minimum of **5,686** pages should be crawled. However, it seems SEMrush is not crawling every single page from the sitemap.

**Comment by amankumarrr** [comment]
Fixed indexing issues from the google console. 


![Image](https://github.com/user-attachments/assets/a8fc6f73-0149-4669-b896-3206a969c03c)

**Figure: Index pages from sitemap**

**Comment by amankumarrr** [comment]
@Marxoz @bradystroud @wicksipedia 

While investigating the Front Door logs, I discovered that 'Browse.aspx' and 'browsexml.aspx' were visited several thousand times. Around the 31st of July, this number dropped to a few hundred, which could explain the significant drop in traffic we've observed in our Google Analytics.  

![Image](https://github.com/user-attachments/assets/9ea55cff-d4dd-47ce-99d3-f8fa640c1309)

**Figure: Front Door logs for the */browse/* pages day wise**

**Comment by amankumarrr** [comment]
I changed the crawl setting from 'Website' to 'Sitemap,' and it seems to have crawled every page in the sitemap now. I initially assumed this change would increase the errors, but surprisingly, the errors have actually decreased.

![Image](https://github.com/user-attachments/assets/84ef6d9a-baa5-441e-92f7-7ad08fef7880)
**Figure: SEMrush audit report after crawling on the sitemap**


After analyzing the difference, I found that when it crawls by source of sitemap, it only crawls only on the pages that are in sitemap.xml whereas when I choose the source as a website it crawls on the depth of 3 to 4 level down the links. 

### Crawling by Sitemap:

    Pros:
      ✅ Crawls only the pages listed in the sitemap.xml.
    Cons:
      ❌ May miss broken links within the site that aren't listed in the sitemap. 
      ❌ May overlook deeper issues as it doesn't crawl beyond the pages explicitly included in the sitemap.
      ❌ Issues with images or assets outside of the sitemap might be ignored.

### Crawling by Website:
    Pros:
     ✅ Crawls up to a depth of 3 to 4 clicks within the site.
     ✅ Identifies detailed issues across multiple layers of the site.
    Cons:
    ❌ Doesn't crawl every page from the sitemap.
    ❌ May miss pages that are only reachable through sitemap links.

**Comment by bradystroud** [comment]
Update

As per my conversation with @amankumarrr, there are 2 main types of Semrush scan – Website and Sitemap

**Website**
-	Starts at the home page, and goes to all the links it can find
-	For some reason, it doesn’t find all SSW.Rules links (even though they are in the menu)
-	Discovers newsletters

**Sitemap**
-	Only looks at links in the sitemap
-	Discovers all rules pages
-	Newsletters are not in the sitemap, so these are not scanned (sitemap generator doesn't know about them - this could be fixed)
-	Maybe there are more pages are missing from the sitemap

Sitemap scan is the best option once we add newsletters to the sitemap. 

To double check there are no other pages missing from the sitemap:

1.	Sitemap scan - Export the list of URLs scanned
2.	Do a website scan
3.	Website scan - Export the list of URLs scanned
4.	Compare it with the list of URLs from the Sitemap scan
5.	See what is missing
6.	Get missing pages added to sitemap

**Comment by amankumarrr** [comment]
I compared the results of the sitemap crawl versus the website crawl and found that approximately 3,337 pages were not crawled when the source was set to "website." The links that were not included in the sitemap crawl are as follows:

- 1,198 pages from /shop, such as: https://www.ssw.com.au/shop?ProductID=STSF01&utm_source=2010 SSW Update December&utm_medium=email&utm_campaign=SSW Mini Scrum in USA
- 499 pages from /events, such as: https://www.ssw.com.au/events?utm_source=Newsletter&utm_medium=Email&utm_campaign=October2011
- 1,568 pages from /ssw/

Based on this analysis, we believe these links are originating from newsletters, which are not currently part of our sitemap. As a result, we've decided to include newsletters within our website’s sitemap. 

This change is being implemented in this PR: https://github.com/SSWConsulting/SSW.Website/pull/3006. Moving forward, we will be using the sitemap as the source for SEMrush crawls, ensuring that all our sites, including /people and /rules, are fully crawled.
